docs/api.md

```diff
@@ -52,6 +52,10 @@ PettingZoo models games as *Agent Environment Cycle* (AEC) games, and thus can s
 
 `action_spaces`: A dict of the action spaces of every agent, keyed by name. This cannot be changed through play or reseting.
 
+`state_space`: The space of a global observation of the environment. Currently, only `butterfly` environments have this attribute implemented. If `state_space` is called in an unsupported environment the following `AttributeError` will raise:
```

Comment:
"Currently, only `butterfly` environments have this attribute implemented." is unnescesary

Reply:
I would mention that not all environments have to implement the method